index.js

```diff
@@ -0,0 +1,13 @@
+var express = require( "express" );
+var app, server, port;
+
+//jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+port = process.env.npm_package_config_port;
+
+//jscs:enable requireCamelCaseOrUpperCaseIdentifiers
+
+app = express();
+app.use ( "/", express.static( "./static" ) );
+server = app.listen( port, function() {
+	console.log( "Listening on " + port + "..." );
```

Comment:
This should output a full URL, like "Listening on http://localhost:3000" to support Ctrl/Cmd clicking it (didn't I mention this before?)